### ISSUE TITLE
Cannot access exist-db via localhost:8080 or any other way! 

### ISSUE THREAD

**Issue by kev1807** [eXist-db/exist#2078]
Hi there, 

After uninstalling and reinstalling I cannot boot access exist-db via localhost:8080 or any other way.
(I guess I played around a little too much for it to remain with the first installation ...) 

exist-db: 4.3.0 ; 

And I still cannot get it to run as a service - has anyone an easy answer? :-D 

best wishes
and thanks in advance!

**Comment by adamretter** [comment]
We are going to need a bunch more information to help you.

Which OS?
Which version of Java?
How did you install eXist-db?
What's present in exist.log?

**Comment by kev1807** [comment]
Log with the last few minutes: 
https://pastebin.com/WsbZPAA4
![grafik](https://user-images.githubusercontent.com/15955790/43368556-53e2da30-935f-11e8-91fa-ac48cfa42e29.png)

Windows 10, 64 bit, 
via the executable jar-file I downloaded a few days back ...

**Comment by dizzzz** [comment]
for sure the 32bit JDK will cause problems....

**Comment by dizzzz** [comment]
Please verify if you have a firewall running, a proxy defined, etc;
according to the log -if it is the correct one- eXist-db is just running fine.

**Comment by pherk** [comment]
http://192.168.0.17:8080/exist is configured.
localhost = 127.0.1.1?

**Comment by dizzzz** [comment]
170.0.0.1 ; eXist-db reports the public IP numbers but indeed, they should work provided there is no firewall...

**Comment by kev1807** [comment]
Hi there, sorry for the long offline-time, I had to work.... sadly, this is not part of my job, so I had to postpone answers for now. 
As I had a clean slate, I reinstalled exist-db, again, I got rid of the JDK (32-bit) and cleaned my PC properly. Now besides being unable to use "run as service" for exist-db it works fine ...
I think this rather is a java-issue there with permissions to perform some task, but I haven't been able to figure one out! 
Suggestions are welcome ... 
I am on 4.3.1 now ... decided it was okay to do this ... :-D 

For the aforementioned issue: thanks for all the input!
